src/commands.js

```diff
@@ -110,7 +110,8 @@ PhotoHandler = async (ctx) => {
     const caption = ctx.message.caption || ""
     const target = ctx.isAngel ? ctx.angel : ctx.mortal
     if (target.isRegistered()) {
-        const fileLink = await ctx.telegram.getFileLink(photos[0].file_id)
+        const len = ctx.message.photo.length - 1
```

Comment:
could you please add a brief comment explaining what this photos (array?) is, eg how many items we expect there are